src/USER-LD/manual/ld_manual.tex

```diff
@@ -0,0 +1,149 @@
+\documentclass[12pt,a4paper]{article}
```

Comment:
Documentation should not be in the source tree. Rather this document should be converted into a file `doc/src/pair_local_density.txt` in a style equivalent to existing files in that folder, plus possibly a file Howto_local_density.txt with additional information, e.g. how to build a complete input deck and its support files.
For latex style typesetting, MathJaX can be used. We plan to convert the documentation to using RST soon (currently there is a tool for conversion) and then will support LaTeX typesetting in all versions of the manual. All explicit image files from `doc/src/Eqs` will be converted to MathJaX in the process, so you can save us some time and effort by doing this directly.